fpdf/fpdf.py

```diff
@@ -1369,20 +1369,49 @@ def regular_polygon(self, x, y, numSides, polyWidth, rotateDegrees=0, style=None
         centerY = y - radius
         # center point is (centerX, centerY)
         points = []
-        i = 1
         for i in range(1, numSides + 1):
             point = centerX + radius * math.cos(
                 math.radians((360 / numSides) * i) + math.radians(rotateDegrees)
             ), centerY + radius * math.sin(
                 math.radians((360 / numSides) * i) + math.radians(rotateDegrees)
             )
             points.append(point)
-            i += 1
         # creates list of touples containing cordinate points of vertices
 
         self.polygon(points, style=style)
         # passes points through polygon function
 
+    @check_page
+    def star(self, x, y, rin, rout, corners, rotateDegrees=0, style=None):
+        """
+        Outputs a regular star with n corners.
+        It can be rotated.
+        It can be drawn (border only), filled (with no border) or both.
+
+        Args:
+            x (float): Abscissa of star's centre.
+            y (float): Ordinate of star's centre.
+            rin (float): radius of internal circle.
```

Comment:
I wonder if `r_in` / `r_out` would not be slightly clearer, and more conform to Python standard `snake_case` naming of variables. What do you think?

Reply:
Absolutely yes! What also about `rotateDegrees `in `rotate_degrees`?

Reply:
Sounds good to me!